community-membership.md

```diff
@@ -1,17 +1,15 @@
 # Community membership
 
-**Note:** This document is in progress
+**Note:** This document is a work in progress
 
-This doc outlines the various responsibilities of contributor roles in
-Kubernetes.  The Kubernetes project is subdivided into subprojects under SIGs.
-Responsibilities for most roles are scoped to these subprojects.
+This document outlines the various responsibilities of contributor roles, divided into SIGs (Special Interest Groups) subprojects for the Kubernetes project.
```

Comment:
I would keep this as-is or reword it differently. IMO the highlight should be to show that responsibilities of contributor roles are scoped to subprojects (SIGs are divided into subprojects).

Reply:
Thanks @nikhita I defer to your judgement - as I need to accumulate a much better understandings of the terminology and phraseology :thumbsup:

Reply:
@marcuspaget Can you also push a commit to revert the change on this line? :pray:

Reply:
Hi @nikhita,

done now (I think!) :)

https://github.com/marcuspaget/community/commit/e8c33c464c1df5c116dc65c9e088c6c0bda8c286

Still got my learning wheels on :)  I was trying to do from command line and got following error - which I guess is a mismatch between new commits and my original fetch:

$ git push origin HEAD
To github.com:marcuspaget/community.git
 ! [rejected]          HEAD -> updating_community_membership (non-fast-forward)
error: failed to push some refs to 'git@github.com:marcuspaget/community.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

I tried a pull and then push but got same error. Still trying to wrap my head around fast-forwards etc :)

In the end just updated in github and that seems to have gone okay. Please advise if still lgtm